wordclock_plugins/time_default/time_dutch.py

```diff
@@ -9,11 +9,11 @@ class time_dutch:
     range(7,11) + range(40,44): VIJF OVER
     range(11,15) + range(40,44): TIEN OVER
     range(28,33) + range(40,44): KWART OVER
-    range(11,15) + range(18,22) + range(35,39): TIEN VOOR HALF
```

Comment:
these fixes refer to this stancil layout? https://github.com/bk1285/rpi_wordclock/blob/master/wordclock_layouts/dutch_10x9/stancil_front.png

If not, I'd recommend a second version of time_dutch

Reply:
There are two Dutch layouts an old 10x9 and a newer 11x10. I thought I committed a 11x10 stencil as well with my Dutch 11x10 code. But I don't see it in the Master. I think FrankXO uses the 11x10 as well. There is also a time_dutch109.py

Reply:
This is/was pull request #90 for Dutch 11x10.
It got lost again in the develop branch I guess.

Reply:
Indeed, layout + code got somehow lost. I'll take care to restore them again (on develop first...)